FF13Rando/bin/data/battlescenes.csv

```diff
@@ -750,7 +750,6 @@ btsc14450,,Archylte Steppe,chset_z023_de8b|chset_z023_def8,
 btsc14451,,Archylte Steppe,chset_z023_de8b|chset_z023_def8,
 btsc11448,,Archylte Steppe,chset_z023_de8b|chset_z023_def8,
 btsc14447,,Archylte Steppe,chset_z023_de8b|chset_z023_def8,
-btsc11457,,Archylte Steppe,chset_z023_de8b|chset_z023_def8,Mission
```

Comment:
ah this must be why mission 15 hasn't been randoed. I'll go find the correct btsc ID

Reply:
Fixed mission 15 on the dev branch